### PR TITLE
OSSMDOC-227: update output from comand oc get smcp

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6-4.11

Issue: https://issues.redhat.com/browse/OSSMDOC-227

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-227/service_mesh/v1x/installing-ossm.html#ossm-control-plane-deploy-cli_installing-ossm-v1x

Eng review - jwendell
QE review - mattmahoneyrh
Peer review - jstickler

### CODE REVIEW

**Review by jwendell — APPROVED** [review]

**Comment by mattmahoneyrh** [comment]
LGTM

**Review by JStickler — CHANGES_REQUESTED** [review]
I checked this against a cluster, and we need to update the output to match the current status.

File: modules/ossm-control-plane-deploy-1x.adoc [hunk 1]
```diff
@@ -109,11 +109,11 @@ $ oc create -n istio-system -f istio-installation.yaml
...
 ----
 +
-The installation has finished successfully when the STATUS column is `InstallSuccessful`.
+The installation has finished successfully when the STATUS column is `ComponentsReady`.
 +
 ----
-NAME            READY   STATUS              TEMPLATE   VERSION   AGE
-basic-install   9/9     InstallSuccessful   default    v1.1      4m25s
+NAME            READY   STATUS              PROFILES   VERSION   AGE
+basic-install   9/9     ComponentsReady     default    v1.1      4m25s
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-control-plane-deploy-1x.adoc]: @JStickler thanks for noticing this; the change has been made and the preview is now available.

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**